atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/SchemaMutationLockTables.java

```diff
@@ -100,10 +100,13 @@ private void createTableWithCustomId(TableReference tableRef) throws TException
                 + "    PRIMARY KEY (key, column1, column2)\n"
                 + ") WITH COMPACT STORAGE\n"
                 + "    AND id = '%s'";
-        CqlQuery query = new CqlQuery(createTableStatement,
-                SafeArg.of("keyspace", keyspace),
-                SafeArg.of("internalTableName", internalTableName),
-                SafeArg.of("cfId", uuid));
+        CqlQuery query = CqlQuery.builder()
+                .safeQueryFormat(createTableStatement)
+                .addArgs(
+                        SafeArg.of("keyspace", keyspace),
+                        SafeArg.of("internalTableName", internalTableName),
```

Comment:
any reason we assume safe here and don't use LoggingArgs like we do elsewhere?

Reply:
I think this is because the log arbitrator works on the basis of schema metadata, while the lock tables are an internal table that isn't a part of a product's atlas schema.

It would probably be nice to push this logic into a part of the log arbitrator so that it handles these cases properly, though